node/src/accountant/payment_adjuster/logging_and_diagnostics/log_functions.rs

```diff
@@ -0,0 +1,228 @@
+    }
+
+    let adjusted_accounts_wallets: Vec<&Wallet> = adjusted_accounts
+        .iter()
+        .map(|account| &account.wallet)
+        .collect();
+    let excluded: Vec<(&Wallet, u128)> = original_account_balances_mapped.iter().fold(
+        vec![],
+        |mut acc, (wallet, original_balance)| {
+            if !adjusted_accounts_wallets.contains(&wallet) {
+                acc.push((wallet, *original_balance));
+            }
+            acc
+        },
+    );
+    let adjusted_accounts_summary =
+        format_summary_for_included_accounts(&original_account_balances_mapped, adjusted_accounts);
+    let excluded_accounts_summary_opt = excluded
+        .is_empty()
+        .not()
+        .then(|| format_summary_for_excluded_accounts(&excluded));
+    vec![
+        Some(adjusted_accounts_summary),
+        excluded_accounts_summary_opt,
+    ]
+    .into_iter()
+    .flatten()
+    .join("\n")
+}
+
+pub fn accounts_before_and_after_debug(
+    original_account_balances_mapped: HashMap<Wallet, u128>,
+    adjusted_accounts: &[PayableAccount],
+) -> String {
+    format!(
+        "\n\
+            {:<length$} {}\n\
+            \n\
+            {:<length$} {}\n\
+            {:<length$} {}\n\
+            \n\
+            {}",
+        "Payable Account",
+        "Balance Wei",
+        BLANK_SPACE,
+        "Original",
+        BLANK_SPACE,
+        "Adjusted",
+        format_brief_adjustment_summary(original_account_balances_mapped, adjusted_accounts),
+        length = WALLET_ADDRESS_LENGTH
+    )
+}
+
+pub fn info_log_for_disqualified_account(
+    logger: &Logger,
+    account: &DisqualificationSuspectedAccount,
+) {
+    info!(
+        logger,
+        "Shortage of MASQ in your consuming wallet will impact payable {}, ruled out from this \
+        round of payments. The proposed adjustment {} wei was below the disqualification limit \
+        {} wei",
+        account.wallet,
+        account
+            .proposed_adjusted_balance_minor
+            .separate_with_commas(),
+        account.disqualification_limit_minor.separate_with_commas()
+    )
+}
+
+pub fn log_adjustment_by_service_fee_is_required(
+    logger: &Logger,
+    payables_sum: u128,
+    cw_service_fee_balance: u128,
+) {
+    warning!(
+        logger,
+        "Total of {} wei in MASQ was ordered while the consuming wallet held only {} wei of MASQ \
+        token. Adjustment of their count or balances is required.",
+        payables_sum.separate_with_commas(),
+        cw_service_fee_balance.separate_with_commas()
+    );
+    info!(logger, "{}", REFILL_RECOMMENDATION)
+}
+
+pub fn log_insufficient_transaction_fee_balance(
+    logger: &Logger,
+    cw_required_transactions_count: u16,
+    txn_fee_required_per_txn_minor: u128,
+    transaction_fee_minor: U256,
+    limiting_count: u16,
+) {
+    warning!(
+        logger,
+        "Transaction fee balance of {} wei is not going to cover the anticipated fee to send {} \
```

Comment:
Let's mention how much balance the user needs to maintain to pass the tx fee barrier. Similar with the above service fee.